### PR TITLE
New Weblate documentation

### PR DESCRIPTION
**Pull request by Noam-Alum**
New Weblate documentation.

### CODE REVIEW

**Comment by bennyvasquez** [comment]
thanks for this one, too, @Noam-Alum! Adding this one to my list for today or tomorrow as well.

File: docs/Help-translating-site.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: 'Help translating site'
+---
+# Help translating the site
+
+The [almaLinux.org websites](https://almalinux.org/) localization and translation is managed by [Weblate](https://hosted.weblate.org/engage/almalinux/) and supported by contributors, the translations that get submitted through Weblate are automatically submitted to the [almaLinux.org repo](https://github.com/AlmaLinux/almalinux.org) as a pull request.
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:6]:
```suggestion
The [almalinux.org website](https://almalinux.org/) localization and translation is managed by [Weblate](https://hosted.weblate.org/engage/almalinux/) and supported by contributors. The translations that get submitted through Weblate are automatically submitted to the [almalinux.org repo](https://github.com/AlmaLinux/almalinux.org) as a pull request.
```

File: docs/Help-translating-site.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+> Pull requests are reviewed by a member of the marketing SIG or another team lead, and merged as appropriate.
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:8]:
```suggestion
Pull requests from Weblate are reviewed by a member of the marketing SIG or another team lead, and merged as appropriate.

If you would like to help contribute to translations, you can do so by following the directions below!
```

File: docs/Help-translating-site.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Create an account
+
+First you need to create a [Weblate](https://hosted.weblate.org/accounts/register/) account, you could use accounts from other service providers or fill in the form shown below to create it.
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:12]:
```suggestion
First you need to create a [Weblate](https://hosted.weblate.org/accounts/register/) account. You could use accounts from other service providers or fill in the form shown below to create it.
```

File: docs/Help-translating-site.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![image](/images/weblate-create-account.png)
+
+## Starting to translate
+
+After creating an account we need to head to [AlmaLinux Foundation Weblate](https://hosted.weblate.org/projects/almalinux/):
+
+![image](/images/weblate-almalinux-foundation.png)
+
+And choose each one of the two and you should be sent to a page similar to the image below.
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:22]:
```suggestion
Choosing each one of the two should send you to a page similar to the image below.
```

File: docs/Help-translating-site.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+![image](/images/weblate-translate-website.png)
+
+Then choose the language you want to translate to, for this example I'll choose [Hebrew](https://hosted.weblate.org/translate/almalinux/website-backend/he/?q=state:%3Ctranslated).
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:26]:
```suggestion
Then choose the language you want to translate to. For this example, I'll choose [Hebrew](https://hosted.weblate.org/translate/almalinux/website-backend/he/?q=state:%3Ctranslated).
```

File: docs/Help-translating-site.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+![image](/images/weblate-send-translations.png)
+
+After translating from English to your chosen language (.e.g Hebrew) press on "Save and continue" and your translation would be sent eventually as a PR to the [almaLinux.org repo](https://github.com/AlmaLinux/almalinux.org) and reviewed by a member of the marketing SIG or another team lead, then merged as appropriate.
```
Comment (bennyvasquez) [inline docs/Help-translating-site.md:30]:
```suggestion
After translating from English to your chosen language (.e.g Hebrew) press on "Save and continue" and your translation will be sent eventually as a PR to the [almalinux.org repo](https://github.com/AlmaLinux/almalinux.org) and reviewed by a member of the marketing SIG or another team lead, then merged as appropriate.
```

**Review by bennyvasquez — COMMENTED** [review]
@Noam-Alum this is GREAT! I have some suggestions that are mostly phrasing and grammar. If you like them, feel free to merge them into your PR and then we can get this merged in!

**Comment by Noam-Alum** [comment]
> @Noam-Alum this is GREAT! I have some suggestions that are mostly phrasing and grammar. If you like them, feel free to merge them into your PR and then we can get this merged in!

Like I said on the blog PR, thank you! I'll merge the comments.

**Review by bennyvasquez — APPROVED** [review]
This is good to go!

**Comment by Noam-Alum** [comment]
> This is good to go!

Thank you!

**Pull request merged**